packages/web/src/components/track/trackTileMessages.ts

```diff
@@ -9,5 +9,6 @@ export const messages = {
   reposted: 'Reposted',
   repostedAndFavorited: 'Reposted & Favorited',
   timeLeft: 'left',
-  loading: 'Loading...'
+  // Hidden behind skeleton
+  loading: '------'
```

Comment:
Do we need a message at all if it's hidden?

Reply:
yeah the span cant be blank :D I was noticing the Loading text was clipping underneath the skeleton, which is why i went with this.